### DISCUSSION TITLE
Highlight activated list item

### DISCUSSION THREAD

**Discussion by JakobVogelsang** [material-components/material-web#5307] [Q&A] [answered]
Hi all,

I want to use the components `md-list` and `md-list-item` to create a custom purpose list. One of the requirements is that the activated list item shall be highlighted somehow. E.g. the way `mwc-list` did with the `activated` property. 

I have looked into the code and see that with type!="text" a `md-focus-ring` is part of the DOM, but in my case it is always disabled. 

Is such a highlighting possible with the two components out of the box?

**Comment by asyncliz** [comment, marked as answer]
Not yet, pre-built selection lists on our on todo list (ha).

You'd need to hook up the logic yourself right now. You should be able to use `background` to give it a highlight color. `<md-menu-item>` does something similar you could reference.

**Comment by JakobVogelsang** [comment]
Thank you very much for the hint!
